ci/jenkins/test-vmc.sh

```diff
@@ -416,6 +427,33 @@ function run_conformance {
     else
         echo "All tests passed."
     fi
+
+    if [[ "$COVERAGE" == true ]]; then
+        rm -rf ${GIT_CHECKOUT_DIR}/conformance-coverage
+        mkdir -p ${GIT_CHECKOUT_DIR}/conformance-coverage
+        collect_coverage
+        tar -zcf ${GIT_CHECKOUT_DIR}/$TESTCASE-coverage.tar.gz ${GIT_CHECKOUT_DIR}/conformance-coverage
+        export CODECOV_TOKEN="a2b1c854-8627-4747-89cb-c312da33227d"
```

Comment:
@lxiaopei @lzhecheng I don't think this token is meant to be public!

can we generate a new one and find a way to pass it through the Jenkins environment

Reply:
The first time this token is introduced into Antrea is here:
https://github.com/vmware-tanzu/antrea/blob/master/codecov.yaml

As for Jenkins, it is easy to store this value as one credential. But I am not sure about this yaml file. @weiqiangt @lxiaopei can its value be stored somewhere safe?

Reply:
Accordingt to the [doc](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets), we can store the secret here. But I don't know who has the right to do this, the settings tab is not on my page.

Reply:
Thanks @tnqn for helping create the secret.
@lxiaopei please use the secret `CODECOV_TOKEN` instead. For more detail, you can refer to the [link](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow).

Reply:
/test-e2e-coverage